src/Installer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Composer\Script\Event;
+use RecursiveIteratorIterator as RIterator;
+use FilesystemIterator as FSIterator;
+use RecursiveDirectoryIterator as DirIterator;
+
+final class Installer
+{
+    public static function postUpdate(Event $event = null): void
+    {
+        self::chmodRecursive('runtime', 0777);
+    }
+
+    private static function chmodRecursive(string $path, int $mode): void
+    {
+        chmod($path, $mode);
+        $iterator = new RIterator(
+            new DirIterator(
```

Comment:
```suggestion
            new DirIterator($path, FSIterator::SKIP_DOTS | FSIterator::CURRENT_AS_PATHNAME),
```
Could you do it in one line?
With short names it does not longer than 120 characters